src/base.js

```diff
@@ -71,8 +71,9 @@ class Base extends EventEmitter {
     );
   }
 
+  // eslint-disable-next-line class-methods-use-this
   isVideoUrl(url) {
-    return url.startsWith(`${this.config.baseUrl}/video/v1/`);
+    return url.startsWith(`/video/v1/`);
```

Comment:
Adding this comment for completeness:

https://github.com/muxinc/mux-node-sdk/pull/94/files#diff-0e0d9d7c1687287978d0e94fda2d4adccec5ab3d6351b790b8891dbc8ca89861L61

This line uses `res.config.url` which has a breaking change in `0.19.1`

https://github.com/axios/axios/blob/master/CHANGELOG.md#0191-jan-7-2020

https://github.com/axios/axios/pull/2391

`res.config.url` used to contain the baseUrl, but now it no longer does. It has the relative path